frontend/src/components/ToolTip/ToolTip.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC } from 'react';
+import { colorPalette } from 'stylesheet';
+import { ToolTip, ToolTipText } from './ToolTip.style';
+
+interface Props {
+  children?: any;
+  toolTipText?: string;
+  color?: string;
+  backgroundColor?: string;
+  invertPosition: boolean;
+  role?: string;
+  id?: string;
+}
+
+const ToolTipGT: FC<Props> = ({
+  children,
+  toolTipText,
+  color = colorPalette.primary1,
+  backgroundColor = colorPalette.primary2,
+  invertPosition,
+  role = 'tooltip',
+  id,
+}) => (
+  <>
+    {id !== undefined && (
+      <ToolTip role={role} id={id}>
+        {children}
+        <ToolTipText
+          invertPosition={invertPosition}
+          color={color}
+          bgcolor={backgroundColor}
+          aria-describedby={id}
+          className="tooltipSpan"
+        >
+          {toolTipText}
+        </ToolTipText>
+      </ToolTip>
+    )}
+    {id === undefined && (
+      <ToolTip role={role}>
+        {children}
+        <ToolTipText
+          invertPosition={invertPosition}
+          color={color}
+          bgcolor={backgroundColor}
+          className="tooltipSpan"
+        >
+          {toolTipText}
+        </ToolTipText>
+      </ToolTip>
+    )}
+  </>
```

Comment:
I'm afraid this condition is totally useless :x
If an attribute is equal to `undefined`, the jsx transformation will not render this attribute.

To clarify my first review, I was thinking of generating an `id` if it is not defined.
Something like
```js
const ToolTipGT: FC<Props> = ({
  children,
  toolTipText,
  color = colorPalette.primary1,
  backgroundColor = colorPalette.primary2,
  invertPosition,
  role = 'tooltip',
  id = `tooltip-${uuid()}`, // `uuid()` helper function doesn't exist, it's for the example
}) => (
```